manifest.json

```diff
@@ -669,6 +669,28 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/windows-nano/1803/helix/amd64",
+              "os": "windows",
```

Comment:
The following special metadata property is needed for these new windows images:

```
"osVersion": "nanoserver-1803",
```

Reply:
thanks Michael, I'm hoping to address all your feedback this afternoon.

Reply:
@MichaelSimons I think I hit most of it except the comment about layers